### PR TITLE
feat: [sc-42715] Extend k6 automation to cover multiple EAs

### PR DESCRIPTION
**Pull request by ad0ll**
Story details: https://app.shortcut.com/chainlinklabs/story/42715

### CODE REVIEW

File: packages/k6/README.md [hunk 1]
```diff
@@ -156,14 +156,16 @@ export MOCK_API_DIR=$(pwd)
...
-For each EA that you want to run, create an .env file in this directory named `[adapter].env`.
+For each EA that you want to run, create a .env file in this directory named `[adapter].env` if your EA requires non-default configuration. .
 E.g.:
 
 ```dotenv
 # coingecko.env
 CACHE_ENABLED=true
 ```
 
+> **Note** > `[adapter].env` can be omitted if your EA doesn't require environment variables to be tested
```
Comment (boxhock) [inline packages/k6/README.md:167]:
```suggestion
> **Note**
> `[adapter].env` can be omitted if your EA doesn't require environment variables to be tested
```

File: packages/k6/automate-start-purge.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+for var in "$@"
+do
+   adapterName=$(echo "$var" | cut -d : -f 1)
+  echo "Attempting to purge containers created for $adapterName"
+
+  echo "Attempting to purge $adapterName"
+   docker container rm "$adapterName-adapter"
+   docker container rm "$adapterName-mock-api"
+   docker network rm "$adapterName-network"
+done
```
Comment (boxhock) [inline packages/k6/automate-start-purge.sh:13]: Adding missing newline

```suggestion
done

```

File: packages/k6/src/test-limits.ts [hunk 3]
```diff
@@ -143,6 +145,7 @@ export default (): void => {
   const before = new Date().getTime()
 
   const config = requests[(vu.idInTest - 1) % uniqueRequests]
+  console.log(config)
```
Comment (boxhock) [inline packages/k6/src/test-limits.ts:148]: Should we keep or remove?

**Review by boxhock — CHANGES_REQUESTED** [review]
Great work on this! Had some minor comments. Not sure why integration tests are failing, could be flaky? Worth trying to re-run and see if result changes

**Review by alejoberardino — APPROVED** [review]

**Review by alejoberardino — APPROVED** [review]

**Pull request merged**